src/main/groovy/org/fundacionjala/gradle/plugins/enforce/tasks/salesforce/deployment/Deploy.groovy

```diff
@@ -44,12 +44,15 @@ class Deploy extends Deployment {
         setupFilesToDeploy()
         logger.debug("${'Creating folder  Deploy at: '}${folderDeploy}")
         createDeploymentDirectory(folderDeploy)
-        if (Util.isValidProperty(project, Constants.FOLDERS_DEPLOY)) {
+        if (Util.isValidProperty(parameters, Constants.FOLDERS_DEPLOY)) {
             deployByFolder()
+            deployToSalesForce()
         } else {
             checkStatusTruncate()
             displayFolderNoDeploy()
             deployTruncateFiles()
+            deployAllComponents()
+            deployToSalesForce()
```

Comment:
maybe you can use the only one instruccion at the end of the if else.
deployToSalesForce() it is defined in the if and else cases